Jamulus.pro

```diff
@@ -125,19 +125,19 @@ win32 {
             message(Using ASIO.)
             message(Please review the ASIO SDK licence.)
 
-            !exists(windows/ASIOSDK2) {
-                error("Error: ASIOSDK2 must be placed in Jamulus windows folder.")
+            !exists(libs/ASIOSDK2/common) {
+                error("Error: ASIOSDK2 must be placed in Jamulus \libs folder.")
```

Comment:
Should be `/libs` or `\\libs`, not `\libs`, I think?

Reply:
Not sure what the Windows convention is?

Reply:
Ok. Changed it. I think `\\` can be used for local paths and network paths. At least it's nothing bad.